website/src/content/docs/general/FAQ.md

```diff
@@ -103,9 +64,7 @@ If you'd like to create a guide for the website, please submit a PR as demonstra
 
 There are two ways to do this:
 
-~~1. Place a chart request bounty using the page on [Open Collective](https://opencollective.com/truecharts-bounties/contribute/request-chart-bounty-72004). Reach out to us on Discord or email orders@truecharts.org if you'd like to discuss further. ~~ Temporarily suspended.
-
-2. Build the app yourself (or have someone build it for you) and request to have it added to the project via a Github PR. You can check the **#development** channel of our [Discord](/s/discord) and create a thread if you'd like information about this.
+2. Build the chart yourself (or have someone build it for you) and request to have it added to the project via a Github PR. You can check the **#development** channel of our [Discord](/s/discord) and create a thread if you'd like information about this.
```

Comment:
```suggestion
Build the chart yourself (or have someone build it for you) and request to have it added to the project via a Github PR. You can check the **#development** channel of our [Discord](/s/discord) and create a thread if you'd like information about this.
```

Only one way currently, and please refer to "use the app template to build it, then submit the results when working". or something